packages/standard-minifier-css/plugin/minify-css.js

```diff
@@ -47,11 +58,27 @@ class CssToolsMinifier {
       cachedResult &&
       cachedResult.depsCacheKey === this.watchAndHashDeps(cachedResult.deps, files[0])
     ) {
+      if (verbose && !this.haveHitAnyCache) {
+        this.haveHitAnyCache = true;
+        setTimeout( () => { // we use a timeout to give all files a chance to finish being minified
+          const stats = [`minifyStdCSS: Total CSS ${this.formatSize(this.totalSize)}`];
+          if (this.totalMinifiedSize!==0) {
+            stats.push(`minified ${this.formatSize(this.totalMinifiedSize)}`);
+            stats.push(`reduction ${Math.round(100-this.totalMinifiedSize*100/this.totalSize)}%`);
+          }
+          console.log(stats.join(", "));
```

Comment:
I would prefer the usage of the Meteor's logging package for this, but this is fine too.

Reply:
Ok, interesting, I have no idea how to do that. Can you please show me some example of code that does it or point me in the right direction?

I'm happy to leave it as is or use the logging feature. I'm personally not expecting verbose modes to be the prettiest thing ever. But it's usually better to do _the proper thing_™ if it's doesn't impact on _getting things done_™.

I had a look at Meteor docs now, but none of these looked appropriate
https://docs.meteor.com/packages/logging.html
https://galaxy-guide.meteor.com/app.html#logs

Reply:
Thank you for textbook answer to my comment. Yes you are correct. Let's keep the console log there for now.